src/packages/core/modal/common/section-picker/section-picker-modal.element.ts

```diff
@@ -32,17 +32,6 @@ export class UmbSectionPickerModalElement extends UmbModalBaseElement<
 			'umbSectionsObserver';
 	}
 
-	#submit() {
-		this.value = {
-			selection: this.#selectionManager.getSelection(),
```

Comment:
I didnt test, just looking at the code, but it seems to me that this is missing now? Or how does this modal set the value of the modal.. well I could have investigated, but it would be good if you had a look at this :-)

Reply:
Good catch. It was removed by mistake. It is back again.